app/controllers/pages_controller.rb

```diff
@@ -1,4 +1,6 @@
 class PagesController < ApplicationController
+  skip_before_action :authenticate_user!, only: :home
```

Comment:
Do we want to have the garden index and show also available to those not logged in?